l2l/optimizers/evolutionstrategies/optimizer.py

```diff
@@ -189,7 +189,7 @@ def post_process(self, traj, fitnesses_results):
             # We need to convert the current run index into an ind_idx
             # (index of individual within one generation)
             traj.v_idx = run_index
-            ind_index = traj.par.ind_idx
+            ind_index = traj.parameters.ind_idx
```

Comment:
Not sure if it is too much to do but maybe having an `individual_index`instead `ind_idx` is much clearer.